spec/Records.tex

```diff
@@ -522,7 +434,7 @@ \subsection{Record Assignment}
 a default assignment operator for each record type \chpl{R} having the signature
 \begin{example}
```

Comment:
Let's remove \begin{example}, \end{example}.

Please verify that PDF looks good after that.